plugins-ui/proto/constraint.proto

```
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+package types;
```

Buf (1.55.1)

Files with package "types" must be within a directory "types" relative to root but were in directory "plugins-ui/proto".

(PACKAGE_DIRECTORY_MATCH)